docs/introduction/monorepo.md

```diff
@@ -49,14 +59,19 @@ also in `composer.json` in the root of monorepo. In monorepo, Shopsys packages a
 * **parameters_monorepo.yml** - overriding of global variables of Shopsys Framework, which makes it possible to run
 Shopsys Framework in monorepo
 
+## Installation of monorepo
+Installation of Shopsys Framework is almost the same as is described in [Installation Using Docker](../installation/installation-using-docker.md),
+but instead of creating a new project based on project_base repository, you have to clone [`shopsys/shopsys`](https://github.com/shopsys/shopsys).
```

Comment:
```suggestion
but instead of creating a new project based on `project-base` repository, you have to clone [`shopsys/shopsys`](https://github.com/shopsys/shopsys).
```